src/AutoBuild.jl

```diff
@@ -912,7 +1015,7 @@ function rebuild_jll_package(obj::Dict;
                              lazy_artifacts::Bool = false,
                              from_scratch::Bool = true)
     if build_version === nothing
-        build_version = BinaryBuilder.get_next_wrapper_version(obj["name"], obj["version"])
+        build_version = BinaryBuilder.get_next_wrapper_version(nothing, obj["name"], obj["version"])
```

Comment:
Let's thread a `universe` kwarg through `rebuild_jll_package` and default it to `nothing`.  This will be useful on Yggdrasil.

Reply:
ok